Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/TypeBindingGenerator.cpp

```diff
@@ -842,7 +842,7 @@ void GenerateFunction_GetBoundClass(FCodeWriter& SourceWriter, UClass* Class)
 	if (Class->ClassGeneratedBy)
 	{
 		// This is a blueprint class, so use Unreal's reflection to find UClass pointer at runtime.
-		SourceWriter.Printf("return FindObject<UClass>(ANY_PACKAGE, TEXT(\"%s\"));", *Class->GetName());
+		SourceWriter.Printf("return LoadObject<UClass>(nullptr, TEXT(\"%s\"), nullptr, LOAD_None, nullptr);", *Class->GetPathName());
```

Comment:
Quick question, if this is a blueprint class how come we don't do any `_C` magic here?